README.md

```diff
@@ -1 +1,7 @@
-# Design-Pattern-Assignment
\ No newline at end of file
+# Design-Pattern-Assignment
+
+1)Abstract Factory Pattern	
```

Comment:
Why is setFeature in an interface if both the subclasses are implementing it? Why not in the base class?
What does unique() function do?